geode-redis/src/distributedTest/java/org/apache/geode/redis/RedisDistDUnitTest.java

```diff
@@ -84,7 +89,62 @@ public static void setup() {
     client2 = cluster.getVM(4);
   }
 
+  class ConcurrentSADDOperation extends ClientTestBase {
+
+    private final Collection<String> strings;
+    private final String key;
+
+    protected ConcurrentSADDOperation(int port, String key, Collection<String> strings) {
+      super(port);
+      this.strings = strings;
+      this.key = key;
+    }
+
+    @Override
+    public void run() {
+      Jedis jedis = new Jedis(LOCALHOST, port, JEDIS_TIMEOUT);
+      for (String member : strings) {
+        jedis.sadd(key, member);
+      }
+    }
+  }
+
   @Test
+  public void testConcurrentSaddOperations_runWithoutException_orDataLoss()
+      throws InterruptedException {
+    List<String> set1 = new ArrayList<>();
+    List<String> set2 = new ArrayList<>();
+    int setSize = populateSetValueArrays(set1, set2);
+
+    final String setName = "keyset";
+
+    AsyncInvocation<Void> remoteSaddInvocation =
+        client1.invokeAsync(new ConcurrentSADDOperation(server1Port, setName, set1));
+
+    client2.invoke(new ConcurrentSADDOperation(server2Port, setName, set2));
+
+    remoteSaddInvocation.await();
+
+    Jedis jedis = new Jedis(LOCALHOST, server1Port, JEDIS_TIMEOUT);
+
+    Set<String> smembers = jedis.smembers(setName);
+
+    assertThat(smembers).hasSize(setSize * 2);
+    assertThat(smembers).contains(set1.toArray(new String[] {}));
+    assertThat(smembers).contains(set2.toArray(new String[] {}));
+  }
+
+  private int populateSetValueArrays(List<String> set1, List<String> set2) {
+    int setSize = 5000;
+    for (int i = 0; i < setSize; i++) {
+      set1.add("SETA-" + i);
+      set2.add("SETB-" + i);
+    }
+    return setSize;
+  }
+
+  @Test
+  @Ignore("GEODE-7905")
```

Comment:
Do these tests still fail intermittently even with the compute fix?

Reply:
so the scope of this story changed-  original branch was dealing with number of commands.  This story now only deals with the SADD command.  We wouldn't expect those tests to pass until the code related to those commands are changed (though we did not explicitly run those testsdone)